chainbase/src/main/java/org/tron/core/service/RewardViCalService.java

```diff
@@ -113,28 +114,30 @@ public long getNewRewardAlgorithmReward(long beginCycle, long endCycle,
                                           List<Pair<byte[], Long>> votes) {
     if (!rewardViStore.has(IS_DONE_KEY)) {
       logger.warn("rewardViCalService is not done, wait for it");
+      try {
+        lock.await();
+      } catch (InterruptedException e) {
+        logger.error("rewardViCalService lock error: {}", e.getMessage());
```

Comment:
If an InterruptedException occurs, the program will continue to execute, but with an additional error log. Is this the expected behavior?

Reply:
Fixed!